File: Mocking/CoffeeMaker/Python/wet_beans_coffee.py
import string

from abstract_coffee_select import *


class WetBeansCoffee(AbstractCoffeeSelect):
    def get_beans(self) -> int:
        return 4

    def grind_beans(self) -> bool:
        if self.__dry_beans() == "beans are dry":
            return True
        return False

    def __dry_beans(self) -> string:
        return "beans are dry"

File: Mocking/CoffeeMaker/CSHARP/CoffeeMaker/WetBeansCoffee.cs
using System;
namespace CoffeeMaker
{
    public class WetBeansCoffee : ICoffeeSelect
    {
        public WetBeansCoffee()
        {
        }

        public int GetBeans()
        {
            return 4;
        }

        public bool GrindBeans()
        {
            if(dry_beans() == "beans are dry")
            {
                return true;
            }

            return false;
        }

        private string dry_beans()
        {
            return "beans are dry";
        }
    }
}
